docs/lago-tutorial/README.rst

```diff
@@ -0,0 +1,223 @@
+   for common enviornments - you are welcome
+   to conribute your on files :)
+
+-  deployment-scripts – This directory contains the deployment scripts
+   for each vm (in our case a script that will
+   install the Jenkins server)
+
+Editing init.json
+^^^^^^^^^^^^^^^^^
+
+The init.json is where we need to describe our environment e.g vms and
+networks, so Lago can create them for us.
+
+//TODO: elaborate about the different features and values that can be
+specified within the file.
+
+
+The network
+^^^^^^^^^^^
+
+Lago allows for a creation of 10 different lans ranging from:
+192.168.200.x to 192.168.209.x
+The subnet will be automatically assigend to the virtual network
+bridge.
+
+Creating the environment
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can use two different approches:
+
+Manual Configurations:
+
+::
+
+    lago init lago-work-dir init.json
+
+(This should be invoked from: /lago-tutorial/scenarios/jenkins)
+
+-  The directory /lago-work-dir will contain the files of our new Lago
+   environment.
+   This directory shouldn't exist before invoking lago init.
+   
+-  --template-repo-path, for specifing the path to the template-repo
+   file. (The templates repository is the place from which the base “qcow2” virtual disks will be copied from).
+   By default Lago will use the file from: http://templates.ovirt.org/repo/repo.metadata\
+   
+-  init.json, the name of the file which describes our environment.
+
+Auto configurations
+
+::
+
+    lago init
+
+This command will use the following default configurations:
+
+-  The command will search for a file named "LagoInitFile" that will be
+   used as the init file.
+-  The workdir will be named ".lago"
+-  As mentioned above, The default repo will be used.
+
+Note: Lago will copy the deployment scripts into the new environment and set up two environment variables.
+For example, if you were at /home/myuser/lago dir, 
+
+- $LAGO_PREFIX_PATH = /home/myuser/lago/.lago/current
+- $LAGO_WORKDIR_PATH = /home/myuser/lago/.lago/
+
+so when using a relative path to the deployment (in the init file), there is no worry that 
+the path will be broken when trying to deploy the vms from a directory that doesn't satisfy the relative path.
+In general, all the other commands of lago can use those variables.
+
+Deploy the VMs
+^^^^^^^^^^^^^^
+
+When using the manual configurations all the commands should be invoked
+from /lago-work-dir.
+
+When using the default workdir you can invoke commands from it's parent,
+or any 'sibiling', for example:
+
+::
+
+    lago init
+    mkdir lolo
+    cd lolo
+    lago status
+
+Now, lets start the vms:
+
+::
+
+    lago start
+
+Or for a specific vm named "server":
+
+::
+
+    lago start server
+
+The command below will run the deployment scripts (from within the
+vms) that were specified in the init.json file.
+
+so, if you used the "manual configurations", the command should be
+invoked from /lago-work-dir.
```

Comment:
This is not true anymore, with the latest lago (still unreleased, but merged) the deploy command will work from anywhere, the scripts will be copied into the prefix 'scripts' dir.